resources/definitions/fdmprinter.def.json

```diff
@@ -2109,8 +2109,8 @@
             {
                 "cool_fan_enabled":
                 {
-                    "label": "Enable Cooling Fans",
-                    "description": "Enables the cooling fans while printing. The fans improve print quality on layers with short layer times and bridging / overhangs.",
+                    "label": "Enable Print Cooling",
```

Comment:
Dict sez; The text "Enable Cooling Fans" in Cura should be changed in "Enable Print Cooling Fans".